test/nl/surf/eduhub_rio_mapper/cli_test.clj

```diff
@@ -1,9 +1,9 @@
 (ns nl.surf.eduhub-rio-mapper.cli-test
   (:require [clojure.test :refer :all]
-            [nl.surf.eduhub-rio-mapper.cli :as cli]))
+            [nl.surf.eduhub-rio-mapper.processing :as processing]))
```

Comment:
Rename de test namespace; als we hier processing testen, moet dit ook `processing-test` zijn.